### ISSUE TITLE
Lack of or Rude Communication

### ISSUE THREAD

**Issue by dstufft** [pypa/packaging-problems#21]
A user has expressed concern that the folks working are either not communicating well (Rude) or are just not communicating at all (lack of responses to threads, issues, etc). How can we foster a welcoming space and ensure nobody feels ignored?

**Comment by gnat** [comment]
Forums are passive and do not clog my inbox: Forums enable me to browse and choose when to pull data without committing my mailbox to it. Mailing lists are inferior because: I have no choice: everything pushed into my mailbox if I subscribe.

Google Groups is a barely acceptable "middle of the road" solution, but as an open source contributor myself, I always think twice before subscribing to yet another mailing list and generally opt not to unless the contribution requires me to do so. I usually unsubscribe shortly after.

**Comment by rsyring** [comment]
See also: https://github.com/python/psf-infra-meta/issues/1

**Comment by merwok** [comment]
Well, we replaced mailing lists with forums now, but I don’t know if that addresses all the points raised.

Documentation was critiqued: is packaging.python.org better? good enough? not sure how to judge here given the vagueness of the initial posts.

Big picture vision and change from many ways to do things to one officially blessed way: hopefully in the near future with a packaging council empowered to steer the community.

@woodruffw is it useful to keep this ticket open?
I think the original issues in 2013 served to write down big and small things, with separate discussions for each, as a first step (state – discuss ‑ act). Then it became the place where non-experts could report their problems, to be directed to docs or specific projects trackers.

I don’t know if the triagers here prefer to keep or close these older, state-true-things-but-not-very-actionable tickets.

**Comment by woodruffw** [comment]
Yeah, I think we can close 🙂 -- the overall state of the PyPA/PyPI/etc. documentation is much better (IMO) in 2025 than it was in 2014, and PyPI now has a full-time support specialist.

(And of course there's this issue tracker, which still tends to be low SNR. But I think that's unavoidable...)
